Sprint-2/errors/1.js

```diff
@@ -2,12 +2,14 @@
 
 // Why will an error occur when this program runs?
 // Try playing computer with the example to work out what is going on
+//
+//Answer: SyntaxError: Identifier 'decimalNumber' has already been declared
```

Comment:
Try playing computer. Step through the function line by line and write down what is happening.

Reply:
Inside the function, the code attempts to redeclare the decimalNumber variable using const
This leads to a conflict because const does not allow the redeclaration of a variable within the same scope. JavaScript throws a SyntaxError when it encounters this.